src/components/ContactInfo/ContactInfo.vue

```diff
@@ -331,6 +335,11 @@ export default class ContactInfo extends Vue {
     this.contactInfo = { ...this.businessContact }
   }
 
+  /** Inform the parent of the current validity state of the component. */
+  @Watch('isEditing', { immediate: true })
+  @Emit('isEditingChange')
+  private emitIsEditing (isEditing: boolean): boolean { return !isEditing }
```

Comment:
It might be me but this doesn't compute. This is emitting an event called "isEditingChange" which is the value of "I'm not editing atm" !?

Reply:
For sure, i see the confusing here. Let me re-word this. 

In truth it is emitting validity, but not doing so clearly. I'll update. 

The jist is, `isEditing === !isValid`

Reply:
Or you could emit the current editing state and let the parent decide it that's valid?

It's another case of "who needs to know what?".

Reply:
For sure, will improve this :)

Reply:
Updated :)